system-test/system-test-script.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#Stopping the script if any command fails
+set -euo pipefail
+export myvar="system-test/logfile"
+cd ..
+ echo "Starting collection roller....."
+bin/collection-roller 2>&1 > $myvar &
+#Getting the process id of collection roller
+pid1=$!
+#Checking if the collection roller service has started
+if [ -z "$pid1" ]
+then
+      echo "Collection Roller is not running"
```

Comment:
This case should cause the application to fail, we don't want to go any further

Reply:
fixed.